app/assets/stylesheets/components/_accordion.scss

```diff
@@ -0,0 +1,109 @@
+.app-c-accordion {
+  &.js-accordion-with-descriptions {
```

Comment:
We use 'js-' prefixed classes for our JavaScript code to bind to, this is to avoid mixing presentation with behaviour

Maybe the JavaScript should add a class with a modifier?

`.app-c-accordion--with-descriptions`

Reply:
You're absolutely right. Can't believe I overlooked that. Will change it to a modifier class, good suggestion.